test/addons/addon.status

```diff
@@ -0,0 +1,19 @@
+prefix addons
+
+[true] # This section applies to all platforms
+
+[$system==aix]
+# https://github.com/nodejs/build/issues/1820#issuecomment-505998851
```

Comment:
these changes were later fixed by https://github.com/nodejs/node/pull/28857

I don't object to skipping them, they add little value, but it pulling in the above would make 8 more similar to later release lines, if you wish.

Reply:
Cherry-picked cleanly